sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/tests/Framework/AuthenticationEventDataTests.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using Xunit;
+using Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.Framework;
+using Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.TokenIssuanceStart.Data;
+using System.Text.Json;
+using System.Text.Json.Nodes;
+
+namespace Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.Tests.Framework
+{
+    /// <summary>
+    /// Tests for AuthenticationEventData
+    /// </summary>
+    public class AuthenticationEventDataTests
+    {      
+        public static string TenantId = "00000000-0000-0000-0000-000000000001";
+        public static string AuthenticationEventListenerId = "00000000-0000-0000-0000-000000000002";
+        public static string CustomAuthenticationExtensionId = "00000000-0000-0000-0000-000000000003";
+
+        public static string BuildDataString(
+            bool hasTenantIdKey = true,
+            bool hasTenantIdValue = true,
+            bool hasAuthenticationEventListenerIdKey = true,
+            bool hasAuthenticationEventListenerIdValue = true,
+            bool hasCustomAuthenticationExtensionIdKey = true,
+            bool hasCustomAuthenticationExtensionIdValue = true)
+        {
+            JsonObject obj = new ();
+
+            JsonObject context = new ();
+            context["protocol"] = "SAML";
+
+            JsonObject dataObj = new ();
+            obj["data"] = dataObj;
+            dataObj["authenticationContext"] = context;
+            dataObj["@odata.type"] = "microsoft.graph.onTokenIssuanceStartCalloutData";
+
+            if (hasTenantIdKey)
+            {
+                dataObj["tenantId"] = hasTenantIdValue ? TenantId : "";
+            }
+            if (hasAuthenticationEventListenerIdKey)
+            {
+                dataObj["authenticationEventListenerId"] = hasAuthenticationEventListenerIdValue ? AuthenticationEventListenerId : "";
+            }
+            if (hasCustomAuthenticationExtensionIdKey)
+            {
+                dataObj["customAuthenticationExtensionId"] = hasCustomAuthenticationExtensionIdValue ? CustomAuthenticationExtensionId : "";
+            }
+
+            return obj.ToJsonString();
+        }
+
+        [Fact]
```

Comment:
Please change this over to NUnit, which is the standard test framework for the current Azure SDK generation.

Reply:
Reason I had used XUnit, it was already being used in 100s of other tests and I was already familiar with it, but I can switch it over.

Reply:
XUnit use is reserved for the legacy (track 1) packages in the repository.   The current generation is standardized on NUnit, so we'll need to change this over.

Reply:
That works.   Please create an issue in the repository to track that work, so that we don't lose awareness and can follow-up if needed.

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/33954